src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Windows/UncommonField.cs

```diff
@@ -136,6 +146,11 @@ internal int GlobalIndex
         private int _globalIndex;
         private bool _hasBeenSet;
 
+        /// <summary>A lazily boxed false value.</summary>
+        private static object s_false;
```

Comment:
You could use [MS.Internal.KnownBoxes.BooleanBoxes](https://github.com/dotnet/wpf/blob/main/src/Microsoft.DotNet.Wpf/src/WindowsBase/MS/Internal/KnownBoxes.cs#L11) which is already used in a bunch of places in this repo.

Reply:
Wasn't aware that existed.  Even better.